### PR TITLE
Add /about, /credits pages with styling & layout

### PR DESCRIPTION
**Pull request by above9Sq**
This PR adds:

- /about and /credits pages & content
- general-content layout & styling block
- update to critical.scss to include new styling block

### CODE REVIEW

File: src/_includes/layouts/general-content.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% extends "layouts/base.html" %} {% block content %}
+
+<div class="[ wrapper wrapper--topbot ]">
+  <article>
+      <div class="[ page-head ] [ page-head--general ]">
+        <h1 class="[ page-title--general ]">{{ title }}</h1>
```
Comment (RainbowArray) [inline src/_includes/layouts/general-content.html]: Probably good to have [ page-title ] on this as well to pull in general `page-title` styles, then override anything you need to override in the `--general` class.

File: src/about.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Plant Information Online provides access to:
+
+* Current Plant and Seed Sources for more than 80,000 plants (largest listing of cultivated plants in North America!) from approximately 1,000 North American firms that will ship plants.
+
+* Contact information and links (where available) for more than 2,700 North American retail and wholesale seed and nursery firms that ship plants.
+
+* Bibliographic details for over 415,000 images of more than 150,000 wild and cultivated plants from around the world in botanical and horticultural books and magazines from 1982 to the present.
+
+*Plant Information Online* is designed for ease of use. It offers cross-referencing of plant names and powerful search functionality. It is updated daily.
+
```
Comment (RainbowArray) [inline src/about.md:14]: `AHL),and Richard T. Isaacson,`

to

`AHL), and Richard T. Isaacson,`

File: src/scss/blocks/_general-content.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.page-head--general {
+  padding-top: #{get-size('300')};
+  padding-bottom: 0;
+}
+
+.page-title--general {
+  line-height: 1.5;
+  @include apply-utility('font', 'base');
```
Comment (RainbowArray) [inline src/scss/blocks/_general-content.scss:7]: Generally I recommend putting `@include` statements at the beginning of the ruleset, followed by other rules, which allows rules to override anything in the mixin.

File: src/scss/blocks/_general-content.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+  // Add tighter flow for elements that follow a heading
+  h1,
+  h2 + *,
+  h3 + * {
+    --flow-space: #{get-size('300')};
+  }
+
+  p {
+    font-size: 1em;
```
Comment (RainbowArray) [inline src/scss/blocks/_general-content.scss]: Prefer `rem` to `em`.

File: src/_includes/layouts/general-content.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{% extends "layouts/base.html" %} {% block content %}
```
Comment (RainbowArray) [inline src/_includes/layouts/general-content.html]: `{% block content %}` should go on the next line.

**Review by RainbowArray — CHANGES_REQUESTED** [review]
Overall this looks really good from a code perspective, a few very minor notes I would recommend changing.

I have not done a visual review yet, but by my eye this looks like it will probably render well. Will double-check once code changes are complete.

Excellent work!

**Comment by above9Sq** [comment]
All fixed. Great suggestions!

I'm now second guessing using a '--general' CSS block/selector, as I'm already needing to tweak it for the contact page. I may turn it into something more specific (--about or --credits) and also use a --contact in a future PR. 

Suggestions always welcome!

**Comment by RainbowArray** [comment]
Tested this locally, and this is looking good. The one thing I would suggest is adding some generic link styles.

Screen Shot 2022-02-10 at 4 16 27 PM


If you look in `scss/blocks/_link.scss`, that is a good source of what to add.

I think you could basically take the styles from `.link` and apply those to an `a` element in general-content, so essentially:

`a {
  color: get-color('quinary-shade');
  text-decoration: none;

  &:hover,
  &:focus {
    text-decoration: underline;
  }
}`

Thanks!

**Comment by above9Sq** [comment]
All done. Great suggestions once again. Looks great!

**Comment by RainbowArray** [comment]
Looks great with the updated changes. Merging. Thanks!

**Pull request merged**